sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/data_transfer_func.py

```diff
@@ -0,0 +1,352 @@
+    :param compute: The compute resource the job runs on.
+    :type compute: str
+    :param inputs: Mapping of inputs data bindings used in the job.
+    :type inputs: dict
+    :param outputs: Mapping of outputs data bindings used in the job.
+    :type outputs: dict
+    :param is_deterministic: Specify whether the command will return same output given same input.
+        If a command (component) is deterministic, when use it as a node/step in a pipeline,
+        it will reuse results from a previous submitted job in current workspace which has same inputs and settings.
+        In this case, this step will not use any compute resource.
+        Default to be True, specify is_deterministic=False if you would like to avoid such reuse behavior.
+    :type is_deterministic: bool
+    :param task: task type in data transfer component, possible value is "copy_data".
+    :type task: str
+    :param data_copy_mode: data copy mode in copy task, possible value is "merge_with_overwrite", "fail_if_conflict".
+    :type data_copy_mode: str
+    """
+    inputs = inputs or {}
+    outputs = outputs or {}
+    component_inputs, job_inputs = _parse_inputs_outputs(inputs, parse_func=_parse_input)
+    # job inputs can not be None
+    job_inputs = {k: v for k, v in job_inputs.items() if v is not None}
+    component_outputs, job_outputs = _parse_inputs_outputs(outputs, parse_func=_parse_output)
+    component = kwargs.pop("component", None)
+    if component is None:
+        component = DataTransferCopyComponent(
+            name=name,
+            tags=tags,
+            display_name=display_name,
+            description=description,
+            inputs=component_inputs,
+            outputs=component_outputs,
+            task=task,
+            data_copy_mode=data_copy_mode,
+            _source=ComponentSource.BUILDER,
+            is_deterministic=is_deterministic,
+            **kwargs,
+        )
+    data_transfer_copy_obj = DataTransferCopy(
+        component=component,
+        name=name,
+        description=description,
+        tags=tags,
+        display_name=display_name,
+        experiment_name=experiment_name,
+        compute=compute,
+        inputs=job_inputs,
+        outputs=job_outputs,
+        task=task,
+        **kwargs,
+    )
+    return data_transfer_copy_obj
+
+
+@pipeline_node_decorator
+def import_data(
+        *,
+        name: Optional[str] = None,
+        description: Optional[str] = None,
+        tags: Optional[Dict] = None,
+        display_name: Optional[str] = None,
+        experiment_name: Optional[str] = None,
+        compute: Optional[str] = None,
+        source: Optional[Union[Dict, Database, FileSystem]] = None,
+        outputs: Optional[Dict] = None,
+        is_deterministic: bool = True,
+        task: Optional[str] = DataTransferTaskType.IMPORT_DATA,
+        **kwargs,
+) -> DataTransferImport:
+    """Create a DataTransferImport object which can be used inside dsl.pipeline as a function.
```

Comment:
should remove `as a function` based on current behavior??

Reply:
removed